src/main/java/seedu/address/model/country/CountryManager.java

```diff
@@ -80,4 +81,23 @@ public void addCountryNote(Country country, Note countryNote) {
 
         countryNotesMap.get(country).add(countryNote);
     }
+
+    /**
+     * Returns the set of notes associated to a particular country.
+     *
+     * @param country The country from which we get CountryNotes.
+     */
+    public Set<Note> getCountryNote(Country country) {
+        return this.countryNotesMap.get(country);
+    }
+
+    /**
+     * Returns all Country Notes currently in the map.
+     *
+     * @return Set of all country notes in the map.
+     */
+    public Collection<Set<Note>> getAllCountryNotes() {
+        return this.countryNotesMap.values();
+    }
```

Comment:
Should return unmodifiable objects. Also, `getCountryNote` doesn't seem to be used, will it be used in a later commit?

Reply:
To be honest, I didn't know the benefits of using unmodifiable objects until your comments on this PR. I've done a cursory [read up on it](https://docs.oracle.com/en/java/javase/12/core/creating-immutable-lists-sets-and-maps.html#GUID-A9B36F89-BA86-41BA-AAE1-ED73A5FC5C6E), thanks! 

`getCountryNote` was put to retrieve note associated to a country, to be used when the `TagNoteMap` shall be initialised from countries. This is related to the commented out initialisation method inside `TagNoteMap`, will be used in a later commit 👍